### PR TITLE
Fix missing internal argument

### PR DESCRIPTION
**Pull request by fabiospampinato**
Fixes this: https://github.com/wooorm/nspell/pull/32#issuecomment-696347510

### CODE REVIEW

**Comment by wooorm** [comment]
Could you add tests that go through these new branches?

**Comment by wooorm** [comment]
@fabiospampinato ping!

**Comment by fabiospampinato** [comment]
@wooorm Hey sorry, I haven't had time for this. Honestly I'm not sure spending time on this is super worth it, like I'm not sure all branches I've introduced are actually reached eventually at this point, but without types I wouldn't feel confident removing some of them.

Anyway I'll dive again in the codebase in a few days, as there's a chance using a `Map` rather than an object may make the library twice as fast again, I'll try to make the test suite happy too.

**Comment by fabiospampinato** [comment]
@wooorm fixed 👍

**Review by wooorm — APPROVED** [review]

File: lib/personal.js [hunk 1]
```diff
@@ -36,7 +38,7 @@ function add(buf) {
       word = word.slice(1)
     }
 
-    self.add(word, model)
+    self.add(word, model, NO_CODES)
```
Comment (wooorm) [inline lib/personal.js:41]: Wait, from the code above, it seems `.add` does not accept codes?

Comment (fabiospampinato) [reply lib/personal.js:41]: The signature of that function should be: `function add(dict, word, codes, options)` so it looks like it does need them? But I'm not too sure honestly, without types and a deeper understanding of how the library works I'm not sure about what each functions needs and does.

Comment (wooorm) [reply lib/personal.js:41]: nspell.add is in the readme, described as taking one or two parameters.

You can test this out locally, by removing it, and then using 'npm test'. While we don't use typescript, we do have tests

Comment (fabiospampinato) [reply lib/personal.js:41]: Right, that extra argument is unnecessary then.

Comment (fabiospampinato) [reply lib/personal.js]: Removed.

**Review by wooorm — APPROVED** [review]

**Pull request merged**